ospf6d/ospf6_interface.c

```diff
@@ -490,19 +495,23 @@ static void ospf6_interface_state_change(uint8_t next_state,
 	}
 	oi->state_change++;
 
+	if (oi->area)
```

Comment:
`oi->area` is accessed without NULL-check later on line 516: `OSPF6_ROUTER_LSA_SCHEDULE(oi->area);`
This code is 17-years old so it looks like this check is not needed.

Reply:
Resolved. 
Got it.